test/examples/src/index.md

```diff
@@ -446,7 +446,7 @@ julia> sqrt(100)
 10.0
 
 julia> sqrt = 4
-ERROR: cannot assign variable Base.sqrt from module Main
+ERROR: cannot assign a value to variable Base.sqrt from module Main
```

Comment:
This is really weird. weird.

In Julia 1.0, the old message `ERROR: cannot assign variable Base.sqrt from module Main` is printed.

In Julia >= 1.5, the new message `ERROR: cannot assign a value to variable Base.sqrt from module Main` is printed (I did not check 1.1 - 1.4).

But we are running our tests in Julia >= 1.6 all the time, so why does this test only start to fail with this PR???

Could it be that for some reason it was not run at all? Or that it was run and produced diff here but for some reason that was ignored?!